src/main/java/org/acra/sender/EmailIntentSender.java

```diff
@@ -66,8 +66,12 @@ private String buildBody(@NonNull CrashReportData errorContent) {
 
         final StringBuilder builder = new StringBuilder();
         for (ReportField field : fields) {
+            String value = errorContent.get(field);
+            if (value != null) {

```

Comment:
My first implementation didn't have the null check, but I got a NullPointerException. I didn't look into it any further, but I think it might be because the "user comment" was null. I've seen that in the email that gets sent. `StringBuilder.append` inserts "null" into the string if the argument is null, so I don't think anything will crash if a null is around.